Podfile

```diff
@@ -47,8 +47,8 @@ target 'WooCommerce' do
 
   pod 'WordPressShared', '~> 1.15'
 
-  pod 'WordPressUI', '~> 1.12.4'
-  # pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS.git', :branch => ''
+  # pod 'WordPressUI', '~> 1.12.4'
+  pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS.git', :branch => 'add/bottom-sheet-accessibility'
```

Comment:
I guess we should hold from merging until `1.12.5` is released right?

Reply:
We can merge this once `1.12.5-beta.1` is released (after merging https://github.com/wordpress-mobile/WordPressUI-iOS/pull/110) and we update this branch to use it. Release managers will handle releasing `1.12.5` in the next code freeze (related: paNNhX-33-p2).

Reply:
I made that update to use the released version of `1.12.5-beta.1` in [0e46d18](https://github.com/woocommerce/woocommerce-ios/pull/6482/commits/0e46d186b9c334a9fb3657ca7dd187e63694d729), so we're good to go now.